origin_ui/src/app/(dashboard)/Sidebar.tsx

```diff
@@ -17,25 +17,44 @@
  ***************************************************************/
 
 import Image from 'next/image'
-import styles from "../../app/page.module.css"
-import {Poppins} from "next/font/google";
+import Link from 'next/link'
+import {Typography, Box} from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import HomeIcon from '@mui/icons-material/Home';
+import BuildIcon from '@mui/icons-material/Build';
 
+import styles from "../../app/page.module.css"
 import PelicanLogo from "../../public/static/images/PelicanPlatformLogo_Icon.png"
 import GithubIcon from "../../public/static/images/github-mark.png"
-import {Typography, Box} from "@mui/material";
 
 export const Sidebar = () => {
 
     return (
         <Box>
-            <div className={styles.header} style={{display: "flex", flexDirection: "column", justifyContent:"space-between", padding:"1rem", top:0, position:"relative", zIndex:"1", overflow: "hidden", height: "100vh"}}>
-                <div style={{display:"flex"}}>
-                    <Image
-                        src={PelicanLogo}
-                        alt={"Pelican Logo"}
-                        width={32}
-                        height={32}
-                    />
+            <div className={styles.header} style={{display: "flex", flexDirection: "column", justifyContent:"space-between", padding:"1rem", top:0, position:"fixed", zIndex:"1", overflow: "hidden", height: "100vh"}}>
+                <div style={{display:"flex", flexDirection: "column"}}>
+                    <Link href={"/index.html"}>
```

Comment:
This doesn't seem to be the correct way to patch the issue. Why do we need to append the `index.html` to call the URLs? Isn't this supposed to be handled by the web server (either Go or Nginx)? For running the origin without Nginx, Go should be able to direct all page traffic to`index.html`

Reply:
I think we agreed the although it isn't great that we have to be explicit in our request this is a fix.